python/python/tests/test_async_client.py

```diff
@@ -1107,6 +1107,19 @@ async def test_spop(self, redis_client: TRedisClient):
         assert await redis_client.spop("non_existing_key") == None
         assert await redis_client.spop_count("non_existing_key", 3) == set()
 
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_sinter(self, redis_client: TRedisClient):
+        key1 = f"{{testKey}}:{get_random_string(10)}"
+        key2 = f"{{testKey}}:{get_random_string(10)}"
+        member1_list = ["a", "b", "c"]
+        member2_list = ["c", "d", "e"]
+
+        assert await redis_client.sadd(key1, member1_list) == 3
+        assert await redis_client.sadd(key2, member2_list) == 3
+        # assert await redis_client.custom_command(["SINTER", key1, key2]) == {"c"}
+        assert await redis_client.sinter([key1, key2]) == {"c"}
+
```

Comment:
Please add a test with a non-existing key
Please add a test with a non-Set key

Reply:
https://github.com/aws/glide-for-redis/blob/main/java/integTest/src/test/java/glide/SharedCommandTests.java#L1255-L1268